xml/System.IdentityModel.Protocols.WSTrust/EndpointReference.xml

```diff
@@ -16,15 +16,15 @@
   <Docs>
     <summary>Represents a wsa:EndpointReference element.</summary>
     <remarks>
-      <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- The wsa:EndpointReference element can be used in wst:OnBehalfOf and wp:AppliesTo elements inside of a wsp:RequestSecurityToken element to specify an endpoint reference.  
-  
- Supports either the [WS-Addressing standard schema namespace](http://schemas.xmlsoap.org/ws/2004/08/addressing) or the [WS-Addressing 1.0 schema namespace](https://www.w3.org/2005/08/addressing) when reading. On write only the [WS-Addressing 1.0 namespace](https://www.w3.org/2005/08/addressing) is supported.  
-  
- For more information about the wsa:EndpointReference element, see the Web Services Addressing - Core specification. For more information about its use in WS-Trust protocols, see the version of the WS-Trust specification that applies to your scenario.  
-  
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The wsa:EndpointReference element can be used in wst:OnBehalfOf and wp:AppliesTo elements inside of a wsp:RequestSecurityToken element to specify an endpoint reference.
+
+ Supports either the WS-Addressing standard schema namespace (`http://schemas.xmlsoap.org/ws/2004/08/addressing`) or the [WS-Addressing 1.0 schema namespace](https://www.w3.org/2005/08/addressing) when reading. On write only the [WS-Addressing 1.0 namespace](https://www.w3.org/2005/08/addressing) is supported.
```

Comment:
This is inconsistent formatting. WS-Addressing isn't using the \[text\]\(url\) format, but WS-Addressing 1.0 is.

Reply:
I just realized why this is the case, because the website that xhemas.xmlsoap.org goes to has a forbidden page. Apparently Microsoft owns xmlsoap.org, and the site is hosted in Azure, so we should track down who is responsible for the site and get it fixed instead of modifying the documentation.

Reply:
@mconnew That's because the first URL leads to a 403 and is flagged in our broken link report.

http://schemas.xmlsoap.org/ws/2004/08/addressing

Reply:
> I just realized why this is the case, because the website that xhemas.xmlsoap.org goes to has a forbidden page. Apparently Microsoft owns xmlsoap.org, and the site is hosted in Azure, so we should track down who is responsible for the site and get it fixed instead of modifying the documentation.

Any idea how to track the owner down?

Reply:
I'm trying to find that out.